src/strategies/arbitrum-bps-of-votable-supply/README.md

```diff
@@ -0,0 +1,16 @@
+# Arbitrum Bps of Votable Supply
+
+- [Overview](#overview)
+- [Options](#options)
+
+
+## Overview
+
+This strategy retrun the number of bps (0.01%) of vote a delegate control, which is used to enforce the 0.01% snapshot proposer requirment specified in [The Constitution of the Arbitrum DAO](https://docs.arbitrum.foundation/dao-constitution) with a basic validation module (minScore=1)
```

Comment:
Maybe add `(basis points)` next to the first time bps is mentioned, common finance term, but not really elsewhere